packages/@react-spectrum/picker/test/Picker.test.js

```diff
@@ -820,23 +820,23 @@ describe('Picker', function () {
             </Picker>
           </Provider>
         );
-  
+
         let picker = getByRole('button');
         expect(picker).toHaveAttribute('aria-haspopup', 'listbox');
 
 
         let span = getByText('(required)');
         expect(span).not.toHaveAttribute('aria-hidden');
-  
+
         let label = span.parentElement;
         let value = getByText('Select an option…');
         expect(label).toHaveAttribute('id');
         expect(value).toHaveAttribute('id');
         expect(picker).toHaveAttribute('aria-labelledby', `${label.id} ${value.id}`);
-  
```

Comment:
@dannify how do you feel about making eslint sensitive to these? they're just noise in our PR's
it'll make the linter an angrier linter, but this is getting ridiculous

Reply:
maybe but also we could crack down on whoever doesn't have this set in their IDE

Reply:
![image](https://user-images.githubusercontent.com/9661127/83581197-fc4a7380-a4f2-11ea-8db2-cd3c3c5bff77.png)

Me fix my IDE now